### PR TITLE
test: add integration tests for put

### PR DESCRIPTION
**Pull request by tmyoda**
*Issue #, if available:*
#143 

*Description of changes:*
This pull request adds integration tests for put command.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by StoneDot — COMMENTED** [review]
Thank you for implementing great tests for the `put` command. To make this pull request more excellent one, could you please consider adding the following tests?

* Check the results of multiple `put` invocations for different primary keys.
* Check the results of multiple `put` invocations for the same primary key.

Since this pull request has the required quality to be merged, I have approved it. If you add any additional tests, please let me know. I will review the changes.

**Review by StoneDot — DISMISSED** [review]

File: tests/put.rs [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+    let mut tm = util::setup().await?;
+
+    let expected = r#"
+    [{
+        "pk": {
+            "S": "42"
+        }
+    }]
+    "#;
+    for aciton in ["put", "p"] {
```
Comment (tmyoda) [inline tests/put.rs:55]: I have updated here to verify the command alias "p"

File: tests/put.rs [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+        &table_name,
+        "scan",
+        "-o",
+        "raw",
+    ]);
+
+    let expected = json!(items).to_string();
+    util::assert_eq_json_ignore_order(get_cmd, &expected);
+    Ok(())
+}
```
Comment (tmyoda) [inline tests/put.rs:442]: I have added 3 test based on your review.
- test_multiple_put_different_pk
- test_multiple_put_same_pk
- test_multiple_put_same_pk_different_sk

**Comment by tmyoda** [comment]
I have added 5 tests based on your review and internal meetings:

From internal meetings.
-  `test_put` (added a test for the command alias "p")

Check the results of multiple put invocations for different primary keys.
- `test_multiple_put_different_pk`

Check the results of multiple put invocations for the same primary key.
-  `test_multiple_put_same_pk`

I have also added these tests to make sure that put replaces the existing item and for different sort key:
-  `test_put_same_pk`
-  `test_multiple_put_same_pk_different_sk`

File: tests/put.rs [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+
+#[tokio::test]
+async fn test_put_same_pk() -> Result<(), Box<dyn std::error::Error>> {
+    let mut tm = util::setup().await?;
+    let table_name = tm
+        .create_temporary_table_with_items(
+            "pk",
+            None,
+            vec![util::TemporaryItem::new(
```
Comment (StoneDot) [inline tests/put.rs:322]: Actually, we no longer do not need to use vec!. We can just pass the array. However, we can modify it later in other pull request.

Comment (StoneDot) [reply tests/put.rs:322]: I think this should be handled in another issue based on current usage of the method in other tests.

**Review by StoneDot — APPROVED** [review]
Thank you for polishing up this pull request. LGTM

**Pull request merged**